### ISSUE TITLE
Unable to replicate results as seen on the paper

### ISSUE THREAD

**Issue by Hsengiv2000** [aireveries/RarePlanes#4]
Hi, I have tried replicating the results as found on the paper. I followed the exact steps mentioned 
![image](https://user-images.githubusercontent.com/46417635/88039951-d3517100-cb7a-11ea-95a1-489d59c2369f.png)

(the above is what the paper states)
```
cfg = get_cfg()

cfg.merge_from_file(model_zoo.get_config_file("COCO-Detection/faster_rcnn_R_50_FPN_3x.yaml"))
cfg.DATASETS.TRAIN = ("train",)
cfg.DATASETS.TEST = ()
cfg.MODEL.WEIGHTS = model_zoo.get_checkpoint_url("COCO-Detection/faster_rcnn_R_50_FPN_3x.yaml") # Let training initialize from model zoo

cfg.SOLVER.IMS_PER_BATCH = 4
cfg.SOLVER.BASE_LR = 0.001
cfg.SOLVER.WARMUP_ITERS = 1000
cfg.SOLVER.MAX_ITER = 70000
#cfg.SOLVER.STEPS = (1000, 1500)
#cfg.SOLVER.GAMMA = 0.05
cfg.SOLVER.MOMENTUM = 0.9
cfg.SOLVER.WEIGHT_DECAY = 0.0001
cfg.MODEL.ROI_HEADS.BATCH_SIZE_PER_IMAGE = 64
cfg.MODEL.ROI_HEADS.NUM_CLASSES = 1

cfg.TEST.EVAL_PERIOD = 500

```
The above is the config in Detectron2.

```
class Trainer(DefaultTrainer):
    @classmethod
    def build_detection_train_loader(cls, cfg: CfgNode):
        return build_detection_train_loader(cfg, mapper=DatasetMapper(cfg, is_train=True, augmentations=[
      T.Resize((800, 800)) , T.ColorJitters(brightness = 0.20, contrast = 0.20 , saturation = 0 , hue=0) , T.RandomCrop(512) 
   ]))
```
I even created a custom trainer class for all the augmentations (the only thing I missed was the pixel wise blurring) but either ways, the results are not even half as good. I would like to know the reason behind the disparity or if im doing something wrong. 

Thanks!

**Comment by thomashossler** [comment]
which experiment are you trying to replicate?

**Comment by Hsengiv2000** [comment]
> which experiment are you trying to replicate?

Training on pure synthetic data.
![image](https://user-images.githubusercontent.com/46417635/88128398-73a1a700-cc08-11ea-80ee-cce27f5a1fff.png)
(Forgot to attach the image above)

**Comment by thomashossler** [comment]
the parameters look correct, the main difference is the model used. We used `COCO-Detection/faster_rcnn_R_50_FPN_1x.yaml`. Can you share your training loss / mAP curves?

**Comment by Hsengiv2000** [comment]
> the parameters look correct, the main difference is the model used. We used `COCO-Detection/faster_rcnn_R_50_FPN_1x.yaml`. Can you share your training loss / mAP curves?

I tried training over the weekend with the 1x model, however there are no improvements in the results. I am suspecting issues with my augmentations (though, they were fine when i visualized). Currently experiencing some tensorboard issues and hence cant share my  loss graphs yet, though will share once I fix them.

Could you share a snippet of how you implemented the augmentations? 

Thank you so much.

**Comment by thomashossler** [comment]
Closed for inactivity
